linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands/ansible/shared.yml

```
@@ -4,57 +4,15 @@
 # complexity = low
 # disruption = low
 
-- name: "Configure excluded (non local) file systems"
-  set_fact:
-    excluded_fstypes:
-      - afs
-      - ceph
-      - cifs
-      - smb3
-      - smbfs
-      - sshfs
-      - ncpfs
-      - ncp
-      - nfs
-      - nfs4
-      - gfs
-      - gfs2
-      - glusterfs
-      - gpfs
-      - pvfs2
-      - ocfs2
-      - lustre
-      - davfs
-      - fuse.sshfs
-
-- name: "Create empty list of excluded paths"
-  set_fact:
-    excluded_paths: []
-
-- name: "Create empty list of suid / sgid binaries"
-  set_fact:
-    suid_sgid_binaries: []
-
-- name: "Detect nonlocal file systems and add them to excluded paths"
-  set_fact: 
-    excluded_paths: "{{ excluded_paths | union([item.mount]) }}"
-  loop: "{{ ansible_mounts }}"
-  when: item.fstype in excluded_fstypes
-
-- name: "Find all files excluding non-local partitions"
-  find:
-    paths: "/"
-    excludes: excluded_paths
-    file_type: file
-    hidden: yes
-    recurse: yes
-  register: found_files
-
-- name: "construct list of suid or sgid binaries"
-  set_fact:
-    suid_sgid_binaries: "{{ suid_sgid_binaries | union([item.path]) }}"
-  when: item.mode is match("2.*") or item.mode is match("4.*")
-  loop: '{{ found_files.files }}'
+- name: Search for privileged commands
+  shell: find / -not -fstype afs -not -fstype ceph -not -fstype cifs -not -fstype smb3 -not -fstype smbfs -not -fstype sshfs -not -fstype ncpfs -not -fstype ncp -not -fstype nfs -not -fstype nfs4 -not -fstype gfs -not -fstype gfs2 -not -fstype glusterfs -not -fstype gpfs -not -fstype pvfs2 -not -fstype ocfs2 -not -fstype lustre -not -fstype davfs -type f -perm -4000 -o -type f -perm -2000 2>/dev/null
```

@vojtapolasek Maybe you missed `fuse.sshfs`?

---

```suggestion
  shell: find / -not -fstype afs -not -fstype ceph -not -fstype cifs -not -fstype smb3 -not -fstype smbfs -not -fstype sshfs -not -fstype ncpfs -not -fstype ncp -not -fstype nfs -not -fstype nfs4 -not -fstype gfs -not -fstype gfs2 -not -fstype glusterfs -not -fstype gpfs -not -fstype pvfs2 -not -fstype ocfs2 -not -fstype lustre -not -fstype davfs -type f \( -perm -4000 -o -type f -perm -2000 \) 2>/dev/null
```
Found another issue related to the precedence of the expression.

---

This affects Bash too:
https://github.com/ComplianceAsCode/content/blob/274e50c0c7b002b514dd03bbd746648dc18cc1a6/shared/bash_remediation_functions/perform_audit_rules_privileged_commands_remediation.sh#L60

---

Suggestion to the fstype checks, maybe this is more readable?
`-not \( -fstype afs -o -fstype ceph -o -fstype cifs -o -fstype smb3 -fstype smbfs -o -fstype sshfs -o -fstype ncpfs -o -fstype ncp -o -fstype nfs -o -fstype nfs4 -o -fstype gfs -o -fstype gfs2 -o -fstype glusterfs -o -fstype gpfs -o -fstype pvfs2 -o -fstype ocfs2 -o -fstype lustre -o -fstype davfs -o -fstype fuse.sshfs \)`